### DISCUSSION TITLE
Training Link? Green Software for Practitioners (LFC131)

### DISCUSSION THREAD

**Discussion by benlogan** [Green-Software-Foundation/discussions#57] [Ask Anything]
We seem to have lost a direct link to our training course from our homepage?

I know the banner was replaced to promote the State Of report, but wasn't that the main CTA for training?

@seanmcilroy29 @adamj89 any ideas, am I missing something - I had a good poke around!

Green Software for Practitioners (LFC131)
https://training.linuxfoundation.org/training/green-software-for-practitioners-lfc131/

**Comment by seanmcilroy29** [comment]
@russelltrow - Can you look into this? Possibly add it to the resource dropdown menu.

**Comment by NAMRATA-WOKE** [comment]
Thanks for notifying us @benlogan. I've also  made sure the course link is more prominent in our wiki. @osamajandali can you please add a CTA button under "What is Green Software" on the hompeage next to "read our definition" to include "Take the course" and link it to: https://training.linuxfoundation.org/training/green-software-for-practitioners-lfc131/
